### ISSUE TITLE
NaN validation loss while training the phase 1

### ISSUE THREAD

**Issue by ThanhNguyenTung** [clks-wzz/FAS-SGTD#16]
Thanks for sharing your ideas and source code. It is interesting.

I re-implement your model in tensorflow 2. While training the validation loss is NaN. Did you encounter this problem? If yes, how did you solve this.

Thanks again.

**Comment by clks-wzz** [comment]
Which "gradient type" do you use here? https://github.com/clks-wzz/FAS-SGTD/blob/master/fas_sgtd_single_frame/util/util_network.py#L57
I recommend that you can try "gradient_type == 'type1'".

**Comment by ThanhNguyenTung** [comment]
I saw that. It does fix the problem. Thank you.
